static/js/reducers/index_test.js

```diff
@@ -265,17 +274,27 @@ describe('reducers', () => {
       let profileWithError = Object.assign({}, USER_PROFILE_RESPONSE, {
         first_name: ''
       });
+      let keysToCheck = [["first_name"]];
+      let messages = {"first_name": "Given name"};
 
       // populate a profile
       store.dispatch(receiveGetUserProfileSuccess(USER_PROFILE_RESPONSE));
       store.dispatch(startProfileEdit());
-      dispatchThen(validateProfile(profileWithError), [UPDATE_PROFILE_VALIDATION]).then(profileState => {
+      dispatchThen(validateProfile(
+        profileWithError,
+        keysToCheck,
+        messages
+      ), [UPDATE_PROFILE_VALIDATION]).then(profileState => {
         assert.deepEqual(profileState.edit.errors, {
           first_name: 'Given name is required'
         });
 
         // also assert that it returns a rejected promise
-        store.dispatch(validateProfile(profileWithError)).catch(() => {
+        store.dispatch(validateProfile(
+          profileWithError,
+          keysToCheck,
+          messages
+        )).catch(() => {
           done();
         });
       });
```

Comment:
Can you add a test to cover validation for an employment tab field? Specifically I want to make sure keys with more than one item are covered somewhere